### DISCUSSION TITLE
프로필 페이지 내 좋아요 한 글 목록

### DISCUSSION THREAD

**Discussion by YuHyun-P** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#247] [Ideas]
![image](https://user-images.githubusercontent.com/96400112/174688041-2d6a1f40-b59b-40fe-9d43-ac614b26a9f6.png)

- [ ] 보여도 괜찮다 ⭕
- [ ] 보여주지 말자 ❌
  - [ ] 내 프로필 페이지일 때만, 좋아요 한 글 목록 보여주자

**Comment by Hyevvy** [comment]
저는 개인적으로는 후자입니당 
그리고 api 특성상 좋아요한 글이 보이려면 Post를 전부 받아와서 id를 매칭시켜서 가져와야하는 걸로 알고 있습니다, 그래서 이것도 구현을 꼭 해야하는 기능인지 한 번 더 생각해봐도 좋을 거 같아요 😳

**Comment by YuHyun-P** [comment]
저는 api랑 상관없이 말하자면, 저희 사이트에서 좋아요 기능은 관심 갖는 프로젝트를 저장하는 기능으로 볼 수 있을 것 같아서 보여주면 좋을 것 같습니다. 내가 좋아요 한 글 목록을 볼 때에는 나중에 보거나 기억하고 싶은 프로젝트를 보관하고, 남이 좋아요 한 글 목록을 볼 때에는 이 사람은 이런 프로젝트에 관심을 갖는구나로 작용할 것 같습니다!
